package/models.py

```diff
@@ -1,5 +1,34 @@
 import os
-import shutil
```

Comment:
We still use the shutil in the displa() func, you went ahead to create another display function and removed the shutil library.

this will break the code